### PR TITLE
Add UI mockups.

### PR DESCRIPTION
**Pull request by joceron**
Added UI mockups.
Added toggl service connection to front end (incomplete).

### CODE REVIEW

File: app/controllers/UIController.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import scala.concurrent.ExecutionContext
+
+/**
+  * This controller is mainly responsible for linking the mockups.
+  * @param messagesApi the messagesAPI for internationalisation.
+  * @param executionContext the execution context for asynchronous execution of program logic
+  */
+class UIController(val messagesApi: MessagesApi)(implicit executionContext: ExecutionContext) extends Controller with I18nSupport {
+  val togglTokenForm = Form(single("togglToken" -> text(maxLength = 20)))
```
Comment (mahyaa) [inline app/controllers/UIController.scala]: this should be changed why 20?

File: app/controllers/UIController.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  val projectName = Form(single("projectName" -> text(maxLength = 20)))
```
Comment (mahyaa) [inline app/controllers/UIController.scala]: also here

File: app/services/impl/ApiTogglService.scala [hunk 3]
```diff
@@ -22,7 +22,10 @@ class ApiTogglService(ws: WSClient)(implicit val ec: ExecutionContext) extends T
   override def getTogglProjects(apiToken: String): Future[List[String]] = {
     //TODO: The workspace ID should be passed to this method.
     // Meanwhile, in order to test change the xxxx to a roper workspace.
-    val request = ws.url("https://www.toggl.com/api/v8/workspaces/xxxx/projects")
+
+    val workspace = getTogglWorkspace(apiToken)
```
Comment (mahyaa) [inline app/services/impl/ApiTogglService.scala]: You made workspace and is not used anywhere

File: app/controllers/UIController.scala (hunk 1 above)
Comment (mariadroman) [reply app/controllers/UIController.scala]: True, this is left from some testing code, and not required at all. Will remove it.

File: app/controllers/UIController.scala (hunk 2 above)
Comment (mariadroman) [reply app/controllers/UIController.scala]: True, this is left from some testing code, and not required at all. Will remove it.

File: app/services/impl/ApiTogglService.scala (hunk 3 above)
Comment (mariadroman) [reply app/services/impl/ApiTogglService.scala]: Fixed.

**Review by mahyaa — APPROVED** [review]

**Pull request merged**